src/ray/core_worker/transport/sequential_actor_submit_queue.cc

```diff
@@ -91,14 +91,7 @@ SequentialActorSubmitQueue::PopAllOutOfOrderCompletedTasks() {
   return result;
 }
 
-void SequentialActorSubmitQueue::OnClientConnected() {
-  // This assumes that all replies from the previous incarnation
-  // of the actor have been received. This assumption should be OK
-  // because we fail all inflight tasks in `DisconnectRpcClient`.
-  RAY_LOG(DEBUG) << "Resetting caller starts at for actor " << actor_id << " from "
-                 << caller_starts_at << " to " << next_task_reply_position;
-  caller_starts_at = next_task_reply_position;
-}
+void SequentialActorSubmitQueue::OnClientConnected() {}
```

Comment:
The sequence number definition changes. Originally, the definition is `ActorCounter - caller_starts_at`. See the following two links for more details. Currently, the definition changes to `ActorCounter`.

* https://github.com/ray-project/ray/blob/ad4e989ebffd7a0d4068dd891be79f032082164c/src/ray/core_worker/transport/sequential_actor_submit_queue.h#L82
* https://github.com/ray-project/ray/blob/ad4e989ebffd7a0d4068dd891be79f032082164c/src/ray/core_worker/transport/sequential_actor_submit_queue.cc#L103-L108

Reply:
A possible follow-up: refactor `SequentialActorSubmitQueue`

Reply:
1/N: https://github.com/ray-project/ray/pull/52249

Reply:
2/N: #52292

Reply:
another follow up: https://github.com/ray-project/ray/pull/52292#discussion_r2041392963

Reply:
#52833

Reply:
#52936